app/components/BlogCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { FunctionComponent } from "react";
+import dayjs from "dayjs";
+import cx from "classnames";
+import { PostOrPage } from "@tryghost/content-api";
+
+type Props = {
+  blogPost: PostOrPage;
+};
+
+const BlogCard: FunctionComponent<Props> = ({ blogPost }) => {
+  const tag = (blogPost.tags || [])[0];
```

Comment:
@deammer I have no idea how tag is displayed at Ghost. For i.e, [this blog](https://learn.codesee.io/codesee-case-studies-distribute-aid-code-review/) has several tags as I checked out on Ghost dashboard & API response, but on [codesee learn](https://learn.codesee.io/), it's not showing any tag.
So, as of now, I'm displaying the tag by the below logic:
1. Getting tags from a post.
2. If the first tag's visibility is not `public` (`internal`), then hide the tags. Otherwise, show the first tag.
Can you confirm me if it's right or wrong?

Reply:
Here's my suggestion: display the first 2 public tags on each blog post. I think that should look nice.

Reply:
Thanks. It was updated.